[deps] App fails to start because of deps conflicts

## description
The app fails to start because react-dom is not found

## how to reproduce
- clone the repo
- npm install (notice the peer dependencies errors)
- npm start
- error: `Module not found: Error: Cannot resolve module 'react-dom'`

## specs
Until we upgrade webseed to use React 0.14 (#6) we cannot use the latest version of react-components

## misc
@gabro pointed out there is a `react-0.13` branch of react-components we can use

--- a/package.json
+++ b/package.json
@@ -50,7 +50,7 @@
   },
   "dependencies": {
     "avenger": "^0.2.6",
-    "buildo-react-components": "github:buildo/react-components",
+    "buildo-react-components": "github:buildo/react-components#react-0.13",
     "debug": "^2.2.0",
     "intl": "^1.0.1",
     "invariant": "^2.2.0",

